Tools/TestWebKitAPI/Tests/mac/DragAndDropTestsMac.mm

```diff
@@ -114,6 +115,13 @@
         return [webView stringByEvaluatingJavaScript:@"imageload.textContent"].boolValue;
     }, 2, @"Expected image to finish loading.");
     EXPECT_EQ(1, [webView stringByEvaluatingJavaScript:@"filecount.textContent"].integerValue);
+
+    TestDraggingInfo *draggingInfo = [simulator draggingInfo];
+    NSArray<NSFilePromiseReceiver *> *filePromiseReceivers = [draggingInfo filePromiseReceivers];
+    EXPECT_EQ(1UL, [filePromiseReceivers count]);
+    NSFilePromiseReceiver *filePromiseReceiver = [filePromiseReceivers objectAtIndex:0];
+    EXPECT_EQ(1UL, [filePromiseReceiver.fileTypes count]);
+    EXPECT_WK_STREQ((__bridge NSString *)kUTTypeGIF, [filePromiseReceiver.fileTypes objectAtIndex:0]);
```

Comment:
```suggestion
    EXPECT_WK_STREQ((__bridge NSString *)kUTTypeGIF, filePromiseReceiver.fileTypes.firstObject);
```

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/73b408dbf7b37d76bc77e7905d7a4d5f0606f5f4